wolox-books/src/app/screens/user/user.component.html

```diff
@@ -1,2 +1,2 @@
 <img src="../../assets/logo-wolox.png" alt="wolox-logo" class="logo">
-<ng-content></ng-content>
+<router-outlet></router-outlet>
```

Comment:
mmm you sure about this logo above?

Reply:
I'm sure, because I have a main router-outlet and this the secondary is for the content of the user component